lib/jekyll/plugin.rb

```diff
@@ -60,10 +60,7 @@ def self.priority(priority = nil)
     #
     # Returns the safety Boolean.
     def self.safe(safe = nil)
-      if safe
-        @safe = safe
-      end
-      @safe || false
+      @safe ||= !!safe
```

Comment:
Maybe this needs a comment? IIRC, I think I tried to do something similar awhile back.